# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/svgfe-dropshadow.yaml
# Commit: a0b3e2669a2d
# Full Hash: a0b3e2669a2dc49c467521cb9b7966ca05e3dbc3
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-05-08 09:49:23
# Regressor Bug: 1824502
# File Overlap Count: 1
# Description:
#   Bug 1824502 - Implement native SVG filter rendering in WebRender r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209646
# ==============================================================================

diff -r d1df40ba7c88 -r a0b3e2669a2d gfx/wr/wrench/reftests/filters/svgfe-dropshadow.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/svgfe-dropshadow.yaml	Wed May 08 04:38:26 2024 +0000
@@ -0,0 +1,27 @@
+---
+root:
+  items:
+    - type: stacking-context
+      bounds: [10, 10, 510, 510]
+      filters:
+        - svgfe: SourceGraphic
+        - svgfe: SourceAlpha
+        - svgfe: dropshadow
+          linear: false
+          subregion: [-20, -20, 320, 320]
+          color: red
+          dx: 20
+          dy: 20
+          std_deviation_x: 10
+          std_deviation_y: 10
+          in: 0
+      items:
+        - type: stacking-context
+          bounds: [10, 10, 150, 110]
+          items:
+            - type: rect
+              bounds: [10, 10, 100, 100]
+              color: [127, 127, 127, 1]
+            - type: rect
+              bounds: [50, 30, 100, 60]
+              color: [192, 192, 192, 1]